src/net_processing.cpp

```diff
@@ -4312,9 +4312,10 @@ void PeerManagerImpl::CheckForStaleTipAndEvictPeers()
 
 void PeerManagerImpl::MaybeSendPing(CNode& node_to, Peer& peer, std::chrono::microseconds now)
 {
-    if (m_connman.ShouldRunInactivityChecks(node_to, std::chrono::duration_cast<std::chrono::seconds>(now).count()) &&
+    if (m_connman.ShouldRunInactivityChecks(node_to, std::chrono::duration_cast<std::chrono::seconds>(now)) &&
         peer.m_ping_nonce_sent &&
-        now > peer.m_ping_start.load() + std::chrono::seconds{TIMEOUT_INTERVAL}) {
+        now > peer.m_ping_start.load() + TIMEOUT_INTERVAL)
+    {
```

Comment:
More consistent to keep the `{` on the same line?

Reply:
I think (and other seem to agree, see https://github.com/bitcoin/bitcoin/issues/21735) that multiline if conditions are hard to read because there is no break between condition and body, so changed it here.

Happy to revert, though.

Reply:
I don't particularly want to get into discussion about code style, it just seemed inconsistent to me (as all other `if` cases have the `{` on the same line), if you have a good reason for it it's fine with me.